### PR TITLE
[New Entry] Dart: strings.md

### LINKED ISSUE #4240: [Entry] Strings: New concept entry `strings.md`
### Reason/inspiration (optional)

We would like a new term entry on the `Strings` term for [Dart](https://www.codecademy.com/resources/docs/dart). The entry should go under the following path `docs/content/dart/concepts/strings/strings.md`.

The entry should describe the term and how it is used within Dart. Also include: 
- Add a "Syntax" section.
- An "Example" section that explains strings in Dart.

### Entry Type (select all that apply)

New Concept Entry for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by a-ogutrax**
Created new entry for Dart: strings.md

### Description

- Added Strings term for Dart

### Issue Solved

- Closes #4240 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/dart/concepts/strings/strings.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  String multiLineString = """
+    This is a multi-line
+    string in Dart using double quotes.
+    It allows you to write
+    text across multiple lines.
+  """;
+  print(multiLineString);
+}
+```
+
```
Comment (Vino-Siva) [inline content/dart/concepts/strings/strings.md:61]: `Shell` output here for the above example using,
\```shell
\`Your code-byte example output`
\```

Comment (a-ogutrax) [reply content/dart/concepts/strings/strings.md:72]: Updated based on comment to include output in specified example @Vino-Siva

File: content/dart/concepts/strings/strings.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  String multiLineString = '''
+    This is a multi-line
+    string in Dart using single quotes.
+    It allows you to write
+    text across multiple lines.
+  ''';
+  print(multiLineString);
+}
+```
+
```
[the remark this answers is not in the document]
Comment (a-ogutrax) [reply content/dart/concepts/strings/strings.md:51]: @Vino-Siva Updated based on comment to include output in specified example

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
The syntax section is invalid @a-ogutrax. In the syntax section you just show how the string can be created that's it no output blocks.

**Review by Sriparno08 — APPROVED** [review]
Looks good! 👍

**Pull request merged**